tests/package_weights.py

```diff
@@ -0,0 +1,99 @@
+'''
+
+Assuming you set it up like this:
+
+Clone the openl3 models branch (this is where we'll put the weights files)
+
+    $ git clone https://... models/
+    $ cd models
+    $ git checkout models
+    $ cd ..
+
+Clone the openl3 code branch and install (this downloads the weights files)
+
+    $ git clone https://... openl3/
+    $ cd openl3
+    $ pip install -e .
+
+If you're upgrading from the v0.2.0 weights to the v0.4.0 weights, you'll want to remove 
+the Spectrogram layers from the weight files. You can do it like this:
```

Comment:
I'm assuming this is for developers, not users? If a user upgrade from v0.3.1 to v0.4.0, will everything "just work"?

Reply:
Yes that was basically a note for Ho-Hsiang. Honestly we could remove the files in that directory. The only thing I think we need to keep in there is the `remove_layers.py`

Reply:
per my other comment I don't mind keeping these in as they document the migration nicely